### PR TITLE
update documentation on controller lifecycle

### PR DESCRIPTION
**Pull request by deitch**
Just to make things easier for future reviewers.

### CODE REVIEW

File: packet/cloud.go [hunk 1]
```diff
@@ -202,6 +202,19 @@ func startNodesWatcher(informer informers.SharedInformerFactory, handlers []node
...
 
+	// what this does:
+	// when you create an informer, you start it by calling informer.Run()
+	// however, it can take some time for the local state to sync up. If you use any methods before
+	// it is completely synced, especially get or list, you can end up missing data. In order to
+	// avoid the issue, you run it in the following order:
+	//
+	// 1. create your informer
+	// 2. informer.Run()
+	// 3. create a slice of sync functionse []cache.InformerSynced. The function on each informer is informer.HasSynced
```
Comment (displague) [inline packet/cloud.go:213]: functionse

**Review by displague — APPROVED** [review]
LGTM - In the future, maybe we can refactor the two uses into one function and simply use these notes as documentation for that function.

File: packet/cloud.go (hunk 1 above)
Comment (deitch) [reply packet/cloud.go:213]: Yeah. In case gollum reads it. "functionse"!

Comment (deitch) [reply packet/cloud.go]: Fixed.

**Pull request merged**